docs/community/modular-meetup-intro.md

```diff
@@ -0,0 +1,83 @@
+# Celestia Modular Meetup Program
```

Comment:
on the category page, the preview text shows up as "Modular Meetup Banner" and this can be resolved by adding a description in the frontmatter. i'll leave a suggestion to show how 😎 

<img width="1036" alt="scrot 2023-05-04 at 1 18 35 PM" src="https://user-images.githubusercontent.com/46639943/236281220-ab9b8d2b-e1e9-4baf-a0ea-ae681adecfb4.png">